solr/solrj/src/java/org/apache/solr/common/util/NamedList.java

```diff
@@ -238,6 +238,14 @@ public int indexOf(String name, int start) {
     return -1;
   }
 
+  /***
+   * Scans the list sequentially beginning at index 0
```

Comment:
nitpick:  javadoc should include clarity that it's the index of the name.  It's not great to require looking at a parameter's name to disambiguate the meaning of a method.